dotnet/src/SemanticKernel.Test/CoreSkills/PlannerSkillTests.cs

```diff
@@ -473,8 +473,9 @@ public MockSkill(ITestOutputHelper testOutputHelper)
         }
 
         [SKFunction("Split the input into two parts")]
+        [SKFunctionName("SplitInput")]
         [SKFunctionInput(Description = "The input text to split")]
-        public Task<SKContext> SplitInput(string input, SKContext context)
+        public Task<SKContext> SplitInputAsync(string input, SKContext context)
```

Comment:
Mostly curious, these kind of things would have been warning in the SemanticKernel.csproj (I thought) -- what was the difference between the two that caused this, is that known?

Reply:
![image](https://user-images.githubusercontent.com/35783676/224099238-801a8e26-bfb0-4aff-a00b-8273bab539bd.png)
![image](https://user-images.githubusercontent.com/35783676/224099332-cb8b1ad9-2063-4c26-a546-15dededbff6e.png)

Ah nevermind, I was just always getting these warnings through my IDE already. I swore product code was producing errors once upon a time but it's not right now.